platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -7,4 +7,269 @@
+            } else {
+                options = $.extend(true, defaultOptions, {
+                    message: arguments[0],
+                    closable: false,
+                    buttonLabel: BootstrapDialog.DEFAULT_TEXTS.OK,
+                    callback: typeof arguments[1] !== 'undefined' ? arguments[1] : null
+                });
+            }
+            if (options.btnOKClass === null) {
+                options.btnOKClass = ['btn', options.type.split('-')[1]].join('-');
+            }
+
+            dialog = new BootstrapDialog({
+                type: options.type,
+                title: options.title,
+                message: options.message,
+                closable: options.closable,
+                draggable: options.draggable,
+                data: {
+                    callback: options.callback
+                },
+                buttons: [{
+                    label: options.btnCancelLabel,
+                    action: function (dialog) {
+                        dialog.close();
+                        modals.splice(modals.length-1);
+                        blocked.splice(blocked.length-1);
+                        if (modals.length > 0) {
+                            modals[modals.length-1].open();
+                        }
+                        if (typeof dialog.getData('callback') === 'function' && dialog.getData('callback').call(this, false) === false) {
+                            return false;
+                        }
+                    }
+                }, {
+                    label: options.btnOKLabel,
+                    cssClass: options.btnOKClass,
+                    action: function (dialog) {
+                        dialog.close();
+                        modals.splice(modals.length-1);
+                        blocked.splice(blocked.length-1);
+                        if (modals.length > 0) {
+                            modals[modals.length-1].open();
+                        }
+                        if (typeof dialog.getData('callback') === 'function' && dialog.getData('callback').call(this, true) === false) {
+                            return false;
+                        }
+                    }
+                }]
+            });
+            this.open(dialog);
+        };
+
+        this.motechAlert = function (msg, title, params, callback) {
+            this.alert({
+                title: jQuery.i18n.prop(title),
+                message: jQuery.i18n.prop.apply(null, [msg].concat(params)),
+                callback: callback
+            });
+        };
+
+        this.motechConfirm = function (msg, title, callback) {
+            this.confirm({
+                title: jQuery.i18n.prop(title),
+                message: jQuery.i18n.prop(msg),
+                callback: callback
+            });
+        };
+
+        this.motechAlertStackTrace = function (msg, title, response, callback) {
+            if( title === null || title === '') {
```

Comment:
@akarczewski This might be a bad idea (so feel free to push back), but would it make sense to build the default titles into the default calls? (meaning this.alert or this.confirm)